Source/JavaScriptCore/b3/testb3_8.cpp

```diff
@@ -1094,6 +1094,10 @@ void testStoreAfterClobberExitsSideways()
     RegisterSetBuilder csrs;
     csrs.merge(RegisterSetBuilder::calleeSaveRegisters());
     csrs.exclude(RegisterSetBuilder::stackRegisters());
+#if CPU(ARM)
```

Comment:
Let's leave a comment referencing the bug to undo this, see https://github.com/WebKit/WebKit/pull/37341/files#diff-cd055880f848bd628c629328b14251f6bce5ed9bd76ff64cf4236b75e4758b18R40